docs/guide/src/pcli/transaction.md

```diff
@@ -170,3 +170,65 @@ You will be prompted a disclaimer which you should read carefully, and accept or
 The replicating market makers tool will then generate a list of positions that you can submit by pressing "y", or reject by pressing "n".
 
 There are other pairs available that you can try this tool on, for example `gm:gn` or `gm:penumbra`.
+
+## IBC withdrawals
+
+<!--
+N.B. These steps were written based on a local relayer setup, transferring packets
+between a local devnet and preview. The plan is to run a public relayer between
+testnet & preview again, starting with Testnet 59 Enceladus. We cannot run the relayer
+until 59, due to pcli incompatibilities between the branches.
+-->
+
+Penumbra aims to implement full IBC support for cross-chain asset transfers. For now, however,
+only withdrawals between Penumbra testnet chains are supported. To exercise this logic,
```

Comment:
Technically withdrawals to any testnet are supported, but we're just not running any relayers other than testnet <> preview

Reply:
Fair point! Overall, I'm struggling to balance the tone between what's technically true and what's comprehensible. Not sure I'm the best judge of either. :upside_down_face: I can rephrase to emphasize that the present restriction—which will change rapidly over time—is due to the operation of relayers.

Reply:
Addressed. Thanks for the review!